src/test/java/com/stripe/functional/StripeResponseTest.java

```diff
@@ -28,7 +26,7 @@ public void testResponseIncluded() throws StripeException {
     final Customer customer = Customer.create(null, requestOptions);
     final Customer retrievedCustomer = Customer.retrieve(customer.getId(), requestOptions);
     final StripeResponse response = retrievedCustomer.getLastResponse();
-    assertThat(response, instanceOf(StripeResponse.class));
+    assertNotNull(response);
```

Comment:
The `instanceOf()` assertion is useless: `response` is declared as a `StripeResponse` so we already know the type at compile-time.